find-phantom-resource.ps1

```
@@ -354,91 +369,144 @@ if ($SubScan) {
-            return "NotFound"
+
+        # Direct REST scan - same reliable method as single-RG mode.
+        # Bypasses JMESPath casing bug and finds phantoms invisible to az resource list.
+        # Paginates via nextLink so large RGs are fully covered.
+        $items = [System.Collections.Generic.List[object]]::new()
+        $nextUrl = "/subscriptions/$subscriptionId/resourceGroups/$rgName/resources?api-version=2024-03-01"
+        $restFailed = $false
+        while ($nextUrl) {
+            $restResult = Invoke-AzRest -Url $nextUrl
+            if ($null -eq $restResult -or $null -eq $restResult.value) {
+                Write-Host " (REST failed, skipped)"
```

PSScriptAnalyzer (1.24.0)

[warning] File 'find-phantom-resource.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)